docs/getting-started/Legacy_ESLint_Setup.mdx

```diff
@@ -84,10 +84,40 @@ ESLint will lint all TypeScript compatible files within the current folder, and
 
 ## Next Steps
 
-We provide a plethora of powerful rules that utilize the power of TypeScript's type information. [Visit _Typed Rules_ for a setup guide](./Typed_Linting.mdx).
-
 If you're having problems getting this working, please have a look at our [Troubleshooting & FAQs](../troubleshooting/FAQ.mdx).
 
+### Additional Configs
+
+We provide two shared configs that we recommend you consider enabling:
```

Comment:
"we recommend you consider" seems like a weird mix of words

Maybe something like
- "we recommend you enable for stricter linting", or
- "you should consider enabling"